### PR TITLE
docs: new quick start

### PR DESCRIPTION
**Pull request by nicecui**
## What's Changed in this PR

*Describe the change in this PR*

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

**Review by zhaoyingnan01 — APPROVED** [review]

File: docs/nightly/en/getting-started/quick-start.md [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+SELECT 
+  ts, 
+  count(error) RANGE '5s' AS num_errors,
+  first_value(error) RANGE '5s' AS first_error, 
+  last_value(error) RANGE '5s' AS last_error,
+  host
+FROM
+  app_logs
+WHERE
+  matches(path, 'api/v1')
```
Comment (fengjiachun) [inline docs/nightly/en/getting-started/quick-start.md:216]: Is it necessary to change `path` column to fulltext string?

File: docs/nightly/en/getting-started/quick-start.md [hunk 2]
```diff
@@ -0,0 +1,370 @@
...
+| 2024-07-11 20:00:15 | host1 |        3500 |
+| 2024-07-11 20:00:20 | host1 |        2500 |
++---------------------+-------+-------------+
+8 rows in set (0.15 sec)
+```
+
+### Full-text search
+
+The latency analysis above indicates that `host1` may be experiencing some issues.
+You can utilize the full-text search `matches(path, 'api/v1')` to identify error logs associated with the specific API `api/v1`:
```
Comment (fengjiachun) [inline docs/nightly/en/getting-started/quick-start.md:204]: It seems that `error` column is the full-text string?
```
CREATE TABLE app_logs (
     ts TIMESTAMP TIME INDEX,
     host STRING,
     path STRING,
     error STRING FULLTEXT,
     PRIMARY KEY (host)
 );
```

File: docs/nightly/en/getting-started/quick-start.md [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+| 2024-07-11 20:00:20 | host1 | GetUser     |    2500 |
++---------------------+-------+-------------+---------+
+5 rows in set (0.01 sec)
+```
+
+You can also use functions when filtering the data.
+For example, you can use `approx_percentile_cont` to calculate the 95th percentile of the latency grouped by the host:
+
+```sql
+SELECT approx_percentile_cont(latency, 0.95) AS p95_latency, host FROM grpc_metrics WHERE ts >= '2024-07-11 20:00:10' GROUP BY host;
```
Comment (waynexia) [inline docs/nightly/en/getting-started/quick-start.md:157]: sugg: format this SQL

File: docs/nightly/en/getting-started/quick-start.md [hunk 4]
```diff
@@ -0,0 +1,370 @@
...
+
+The latency analysis above indicates that `host1` may be experiencing some issues.
+You can utilize the full-text search `matches(path, 'api/v1')` to identify error logs associated with the specific API `api/v1`:
+
+```sql
+SELECT 
+  ts, 
+  count(error) RANGE '5s' AS num_errors,
+  first_value(error) RANGE '5s' AS first_error, 
+  last_value(error) RANGE '5s' AS last_error,
```
Comment (waynexia) [inline docs/nightly/en/getting-started/quick-start.md:211]: This doesn't look like a real use case. Who would care about "first log in every 5 seconds"?

Comment (waynexia) [reply docs/nightly/en/getting-started/quick-start.md:211]: I'd suggest not applying `RANGE` on log table but metric table. For log table simply filtering should be enough. Aggregation on log text doesn't make sense.

File: docs/nightly/en/getting-started/quick-start.md [hunk 5]
```diff
@@ -0,0 +1,370 @@
...
+
+Before proceeding, please ensure you have [installed GreptimeDB](./installation/overview.md).
+
+This guide will walk you through creating a metric table and a log table, highlighting the core features of GreptimeDB.
+
+
+## Connect to GreptimeDB
+
+If your GreptimeDB instance is running on` 127.0.0.1` with the default port `4000`,
+you can connect to it using either a MySQL or PostgreSQL client:
```
[the remark this answers is not in the document]
Comment (nicecui) [reply docs/nightly/en/getting-started/quick-start.md:11]: I'll change the port to 4002

File: docs/nightly/en/getting-started/quick-start.md [hunk 6]
```diff
@@ -0,0 +1,370 @@
...
+  metrics.ts,
+  p95_latency, 
+  coalesce(num_errors, 0) as num_errors,
+  logs.first_error,
+  logs.last_error,
+  metrics.host
+FROM 
+  metrics 
+  LEFT JOIN logs ON metrics.host = logs.host 
+  AND metrics.ts = logs.ts
```
[the remark this answers is not in the document]
Comment (killme2008) [reply docs/nightly/en/getting-started/quick-start.md:275]: We can remove the `first_value` and `last_value` of logs, but keep the `count`, it makes sense.

After we find the time point of error, we can query the logs filtered by the time window.

**Comment by nicecui** [comment]
@killme2008 @waynexia @fengjiachun PTAL

File: docs/nightly/en/getting-started/quick-start.md [hunk 7]
```diff
@@ -0,0 +1,368 @@
...
+```shell
+mysql -h 127.0.0.1 -P 4002
+```
+
+Or
+
+```shell
+psql -h 127.0.0.1 -p 4003 -d public
+```
+
```
Comment (killme2008) [inline docs/nightly/en/getting-started/quick-start.md:21]: Do we need to add a note that GreptimeDB supports many protocols, but for quick start, we are using SQL for the demo?

File: docs/nightly/en/getting-started/quick-start.md [hunk 8]
```diff
@@ -0,0 +1,368 @@
...
+## Create tables
+
+Suppose you have a table named `grpc_latencies` that stores the gRPC latencies of your application.
+The table schema is as follows:
+
+```sql
+CREATE TABLE grpc_latencies (
+  ts TIMESTAMPNANOSECOND TIME INDEX,
```
Comment (killme2008) [inline docs/nightly/en/getting-started/quick-start.md:29]: Why use the `nanosecond` precision? Looks like unnecessary.

File: docs/nightly/en/getting-started/quick-start.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (killme2008) [reply docs/nightly/en/getting-started/quick-start.md:273]: Why not use CTE? It's more clear.

File: docs/nightly/en/getting-started/quick-start.md (hunk 8 above)
Comment (nicecui) [reply docs/nightly/en/getting-started/quick-start.md:29]: This is to introduce Influxdb writing data in Dashboard below, which needs TIMESTAMPNANOSECOND data type.

File: docs/nightly/en/getting-started/quick-start.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (nicecui) [reply docs/nightly/en/getting-started/quick-start.md:273]: what is CTE?

[the remark this answers is not in the document]
Comment (killme2008) [reply docs/nightly/en/getting-started/quick-start.md:273]: WITH .... SELECT

https://learnsql.com/blog/what-is-common-table-expression/

File: docs/nightly/en/getting-started/quick-start.md (hunk 8 above)
Comment (nicecui) [reply docs/nightly/en/getting-started/quick-start.md]: changed to use TIMESTAMP here, and used a new table for influxdb writing in Dashboard below.

File: docs/nightly/en/getting-started/quick-start.md (hunk 7 above)
Comment (nicecui) [reply docs/nightly/en/getting-started/quick-start.md:24]: have added a line to note mutiple protocols.

**Comment by nicecui** [comment]
@killme2008 @fengjiachun PTAL

File: docs/nightly/en/getting-started/quick-start.md [hunk 9]
```diff
@@ -0,0 +1,382 @@
...
+- `method_name`: The name of the RPC request method. It is a tag column.
+- `latency`: The latency of the RPC request.
+
+Additionally, there is a table `app_logs` for storing application logs:
+
+```sql
+CREATE TABLE app_logs (
+  ts TIMESTAMP TIME INDEX,
+  host STRING,
+  api_path STRING FULLTEXT,
```
Comment (fengjiachun) [inline docs/nightly/en/getting-started/quick-start.md:52]: Is it still necessary to use `FULLTEXT`? Maybe making `log` `FULLTEXT` is enough.

**Review by fengjiachun — APPROVED** [review]
LGTM

**Review by killme2008 — APPROVED** [review]
LGTM. Do the diagrams need to be updated?

File: docs/nightly/en/getting-started/quick-start.md (hunk 9 above)
Comment (killme2008) [reply docs/nightly/en/getting-started/quick-start.md:52]: I believe that's necessary. You might want to query APIs inaccurately.

File: docs/nightly/en/getting-started/quick-start.md (hunk 8 above)
Comment (killme2008) [reply docs/nightly/en/getting-started/quick-start.md]: Influxdb line protocol supports creating tables automatically. You never have to create the tables yourself when demonstrating the InfluxDB line protocol.

**Pull request merged**